CRM/Donrec/Lang/De/De.php

```diff
@@ -66,7 +66,28 @@ public function currency2word($currency, $quantity) {
         return parent::currency2word($currency, $quantity);
     }
   }
-
+    /**
+     * Get a spoken word representation for the decimal unit of the given currency
+     *
+     * @param string $currency currency symbol, e.g. 'EUR' or 'USD'
+     * @return string          spoken word for the decimal unit, e.g. 'Cent' or 'Rappen'
+     */
+    public function currencyDecimal2word($currency) {
+        switch ($currency) {
+            case 'EUR':
+            case 'USD':
+            case 'CAD':
+            case 'AUD':
+                return 'Cent';
+            case 'GBP':
+                return 'Penny';
+            case 'CHF':
+                return 'Rappen';
+            case 'JPY':
+            default:
+                return '';
```

Comment:
Although for *Cent* and *Rappen* the German plural form is the same as the singular, this is not true for *Penny* (plural *Pence* or *Pennys*), so this function needs to be aware of the amount and return plural forms where necessary.